orion/controllers/programs/programs_test.go

```diff
@@ -12,8 +12,9 @@ func TestExample(t *testing.T) {
 }
 
 func TestCheckProgram(t *testing.T) {
-  isValid := CheckValidProgram()
-  if (isValid == false) {
-    t.Errorf("Check was incorrect, got: %t, expected: %t.", isValid, false)
+  // TODO: add a program to check
+  isValid := CheckValidProgram(program)
```

Comment:
this will probably error because `program` is not initialized, right?
Create a new Program{} and feel free to give it fake values to test if `CheckValidProgram()` works as intended.
You can do
```
program := new Program{name: "asdf", grade1: 1, grade2: 2, ... }
```